dpti/equi.py

```diff
@@ -600,6 +601,35 @@ def post_task(iter_name, natoms=None, is_water=None):
     return info_dict
 
 
+def run_task(task_name, machine_file):
+    task_dir_list = [task_name]
+    work_base_dir = os.path.join(os.getcwd())
```

Comment:
**Suggestion:** Consider using `os.path.abspath()` instead of `os.getcwd()` to get the absolute path of the current directory. This will ensure that the path is always correct, regardless of where the script is run from. [enhancement]
```suggestion
    work_base_dir = os.path.abspath('.')
```